src/components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { StateContext } from '../../reducer/store';
+
+export const Notifications: React.FC = () => {
+  const {
+    notificationMessage,
+    notificationType,
+    notificationData,
+  } = useContext(StateContext);
+
+  return (
+    <div
+      className={classNames(`notification ${notificationType}`)}
+      data-cy={notificationData}
```

Comment:
It's unclear what data-cy attribute is for. If it's for testing purposes, it would be better to use more descriptive name.